utils/src/ooinstall/cli_installer.py

```diff
@@ -920,7 +902,7 @@ def uninstall(ctx):
 @click.option('--latest-minor', '-l', is_flag=True, default=False)
 @click.option('--next-major', '-n', is_flag=True, default=False)
 @click.pass_context
-# pylint: disable=bad-builtin,too-many-statements
+# pylint: disable=too-many-statements
```

Comment:
Should we specify a specific version of pylint to avoid newer versions changing this type of behavior on us unexpectedly?